Essentials/src/plugin.yml

```diff
@@ -5,7 +5,7 @@ main: com.earth2me.essentials.Essentials
 version: ${full.version}
 website: http://tiny.cc/EssentialsCommands
 description: Provides an essential, core set of commands for Bukkit.
-softdepend: [Vault]
+softdepend: [Vault, PlaceholderAPI]
```

Comment:
https://github.com/EssentialsX/Essentials/issues/1945
You once said you don't want to add it as sofedepend.
Will we remove this?

Reply:
Oops, forgot about that. Still need to implement handling of PAPI loading after EssentialsX though before we do that.

Reply:
I will leave that part to you then :)